CHANGELOG.md

```diff
@@ -1,12 +1,31 @@
 # App Center SDK for React Native Change Log
 
-## Version 1.10.0 (Under development)
+## Version 1.10.0
 
 ### Fixes
 
-- Fix lint issue on modern projects using latest react-native versions [#451](https://github.com/Microsoft/AppCenter-SDK-React-Native/issues/451).
-- Use latest Firebase version [#365](https://github.com/Microsoft/AppCenter-SDK-React-Native/issues/365).
-- [Android] Fix a bug which prevents ErrorAttachmentLog from being send if fileName is not specified.
+### AppCenter
+
+- **[Android]** Fix lint issue on modern projects using latest react-native versions [#451](https://github.com/Microsoft/AppCenter-SDK-React-Native/issues/451).
+- **[iOS]** Fix an issue where concurrent modification of custom properties was not thread safe.
+- **[iOS]** Validating and discarding Not a Number (NaN) and infinite double values for custom properties.
+- **[iOS]** Use standard SQL syntax to avoid affecting users with custom SQLite libraries.
+- **[iOS]** Get database page size dynamically to support custom values.
+
+### AppCenterCrashes
+
+- **[Android]** Fix a bug which prevents attachments from being sent if file name is not specified.
+- **[Android]** Fix Preventing stack overflow crash while reading a huge throwable file.
+- **[iOS]** Do not force crash macOS application on uncaught exception. If you need this behavior you can set the special flag yourself:
```

Comment:
This is macOS only while there is no macOS support in RN SDK. No need to mention.

Reply:
addressed